internal/lm/fabric.go

```diff
@@ -142,18 +143,16 @@ func getImexDomainID(r io.Reader) (string, error) {
 }
 
 func generateContentUUID(seed string) string {
-	// nolint:gosec
-	rand := rand.New(rand.NewSource(hash(seed)))
+	r := rand.NewChaCha8(hash(seed))
```

Comment:
Would it make sense to use something like:
```
uudi.NewSHA1("", []bytes(seed)).String()
```
to replace all this logic here? We can decide if we want to make the "namespace" configurable.

Reply:
Sure, I am all for removing code and making it simpler